chingu-voyage4/Geckos-Team-32
# Footer

## Purpose
* Review @odelavia 's code for the `navbar` and `footer` branch
* Make necessary changes and offer future suggestions

## Approach/Fixes
#### Navbar
* All classes should be labeled as `className` since `class` is a reserved word in react
* The `<label>` element's attribute `for` needs to be `htmlFor` in react

#### Footer
* All classes should be labeled as `className` since `class` is a reserved word in react
* Code for copyright needed a semicolon (`&copy;`)
* Changed the hover color for instagram since it was hard to see (temporary change)
* Added `underline` for the links on hover (temporary change)
* When using font-awesome, your `.fab` class is not being called at all I don't think. I'm pretty sure with the new font-awesome update the icons get converted to SVG so their class names aren't manipulated properly through css as they were before. Instead try wrapping them in an element first and styling that (Ex: `<a><i className="fab fa-facebook"></i></a>`)

#### MISC.
* When using SCSS, re-used variables (such as colors, spacing/sizes, etc.) can be saved in the `_variables.scss` file and referenced in separate .scss files
* Took the font links out of the HTML file and instead imported them in the `_variables.scss` file
* You added the font `oswald` but it was never used. I kept it in the scss file for you in case you wanted to use it later, but if not, we should remove it

## Notes on previous commits
* For some reason when you pulled your code, I don't think you merged it properly because there were two index.html files, two bundle.js files, and the file structure was a bit wonky. I cleaned it up and hopefully caught all the differences so this pull request should be all good!
* When making separate branched for separate features, I would suggest keeping their commits separate as well (meaning the footer branch shouldn't include commits from the navbar branch to avoid any merge conflicts; all code for specific branches should be separate)

## Testing
* Download this specific branch to your local machine
* Run the app through `npm run dev` and make sure all the changes have been made!
* Check that the copyright symbol is showing
* Check that the footer links show underline on hover
* Check that the instagram icon shows ~yellow on hover
* Check that the imported fonts are rending properly
* Check that the font-awesome icons are showing up
